meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/apalis-imx6/boot.cmd

```diff
@@ -1,28 +1,17 @@
 # Default to ixora-v1.1, which is the carrier board we officially support
 ## Careful when changing to another dtb as the mmcblk order might differ
 setenv fdt_file_r imx6q-apalis-ixora-v1.1.dtb
+setenv fdt_file ${fdt_file_r}
```

Comment:
Maybe just update the line above this one instead?

Reply:
fixed